pyproject.toml

```diff
@@ -8,6 +8,85 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+astroid = "2.3.3"
```

Comment:
You do not need all of these here!
pyproject.toml does not need to include nested dependencies, and indeed *shouldn't*. And it looks like a lot of these aren't even nested dependencies, they are just packages that are totally unused by this project (did you get the list by doing pip freeze on your global python?)

You should just list python, flask, python-dotenv, requests, and python-dateutil.

Delete everything else.

FYI you shouldn't need to update this file manually. Whenever you add a new dependency to the project, use `poetry add <package name>`. That will update pyproject.toml and also install the package.

Reply:
Agreed - fixed